benchmarks/domains/synthetic_2C1D_1C.py

```diff
@@ -0,0 +1,115 @@
+"""Synthetic function with two continuous and one discrete input."""
+
+from uuid import UUID
+
+import numpy as np
+from numpy import pi, sin, sqrt
+from pandas import DataFrame
+
+from baybe.campaign import Campaign
+from baybe.parameters import NumericalContinuousParameter, NumericalDiscreteParameter
+from baybe.recommenders.pure.nonpredictive.sampling import RandomRecommender
+from baybe.searchspace import SearchSpace
+from baybe.simulation import simulate_scenarios
+from baybe.targets import NumericalTarget, TargetMode
+from benchmarks.definition import Benchmark
+from benchmarks.definition.config import (
+    DEFAULT_RECOMMENDER,
+    RecommenderConvergenceAnalysis,
+)
+
+
+def lookup(z: np.ndarray, x: np.ndarray, y: np.ndarray) -> np.ndarray:
```

Comment:
I'd still like to extend this a little bit by turning domains into classes (or similar) with defined attributes that can be accessed to query information about the domain, like boundaries etc. But this is orthogonal to @fabianliebig's further developments, so we can leave this to another PR. Still, any opinion on this?

Reply:
do we need to include random seed setting/storage here anywhere?

Reply:
not for the current one but potentially for others. But that probably belongs to the benchmark setting, I'd say. So in principle, this would already be fully supported. Unless you have a reason why it would need to be controlled from the outside?

Reply:
Can you elaborate on why you want to have this?

Reply:
I dont think it needs to be controllable, I'd prefer if it was explicitly set and remembered (for all benchmarks, the ones that are not affected will not complaing about the sed being set either)
One could make it an `optional` setting with default `1337` in the same structure that stores `n_mc_runs` etc. It would be set directly before the callable is called

Its mostly for reproducibility, imagine eg a case where results look strange, you might want to reproduce that exaclty for investigation, that might not be possible if the random seed is not explicitly set and remembered